Merge/The Merge: Consensus API design space.md

```diff
@@ -0,0 +1,523 @@
+# The Merge: Consensus API design space
+### Meeting Date/Time: Thursday, Aug 26 at 13:00 UTC
+### Meeting Duration: 60 minutes
+### [Github Agenda](https://github.com/ethereum/pm/issues/378)
+### [Video of the meeting](https://youtu.be/il0nha1HSiE)
+### Moderator: Mikhail Kalinin
+### Notes: Alen (Santhosh)
+
+## Decisions Made / Action items
+| Decision Item | Description | Video ref |
+| ------------- | ----------- | --------- |
+|1 |Discussion about the coinbase might be overridden by the execution client or not to be taken up in the Discord for further discussion | [36.13](https://youtu.be/il0nha1HSiE?t=2173)|
+|1.1 |Usage of the safehead, we need to decide is that the unsafe head is at the safe head or is that finalized safe head? I think is the best option here because it's very close to the tip, but it's also very safe. And where is finalized is potentially pretty old and we don't want to give user block. That's, you know, several minutes old and unsafe is unnecessarily risky to the user. | [50.04](https://youtu.be/il0nha1HSiE?t=3004)|
```

Comment:
```suggestion
|8.2 |Usage of the safehead, we need to decide is that the unsafe head is at the safe head or is that finalized safe head? I think is the best option here because it's very close to the tip, but it's also very safe. And where is finalized is potentially pretty old and we don't want to give user block. That's, you know, several minutes old and unsafe is unnecessarily risky to the user. | [50.04](https://youtu.be/il0nha1HSiE?t=3004)|
```

Reply:
We could add the summary of the decision (if any).